README.md

```diff
@@ -1,2 +1,38 @@
 # testimonials
 Backend for testimonial site
+
+# When running for the first time:
```

Comment:
The instructions seems very local environment specific. While this works, should we also have something specific to docker?